Upstream-Status: Inappropriate [OpenXT]

client.c uses isspace() without including ctype.h. GCC 15 treats
the implicit declaration as an error.

--- a/src/client.c
+++ b/src/client.c
@@ -16,6 +16,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+#include <ctype.h>
 #include "project.h"
 
 
